pkg/common/kubernetes/informers.go

```diff
@@ -30,17 +30,30 @@ func noResyncPeriodFunc() time.Duration {
 	return 0
 }
 
-var signalHandler <-chan struct{}
+var (
+	signalHandler   <-chan struct{}
+	informerFactory informers.SharedInformerFactory
+)
 
 // NewInformer creates a newk8s client based on a service account
-func NewInformer(client clientset.Interface) *InformerManager {
+func NewInformer(client clientset.Interface, singleWatcher bool) *InformerManager {
```

Comment:
Do we need the singleWatcher toggle? In what scenario would we _not_ want a single informer for all lister/watchers?

Reply:
Yea, we don't have a need for it currently. Was just trying to preserve the option. Only reason for it would be having multiple client connections based on individual service accounts.

If you want to remove it and tackle it when we get there, that cool. Just let me know.